### DISCUSSION TITLE
Actualy value of legendary when whichkey and telescope exist?

### DISCUSSION THREAD

**Discussion by airtonix** [mrjones2014/legendary.nvim#361] [General]
If we're using whichkey and telescope, what's the value of legendary over telescope keymaps picker? 

```lua
return {
  n = {
    ["<C-p>"] = { ":Telescope keymaps<cr>", noremap = true }
  }
}
```

![image](https://user-images.githubusercontent.com/61225/233816722-e3e93058-3750-44d9-9963-a65a72d50a96.png)

Considering that telescope just taps into the core nvim api to get all the keymaps, so whichkey groups are properly presented. (legendary leaves out the group name)

Is it that legendary also shows autocommands, commands and keymaps in the one searchable list?

**Comment by mrjones2014** [comment]
Legendary shows keymaps, commands, autocmds, and even arbitrary Lua functions you can execute, all in the same picker. The UX is also more uniform and polished IMO.

> so whichkey groups are properly presented

In `legendary.nvim` they are created as `legendary.nvim` item groups.

If you don't find value in it then don't use it :slightly_smiling_face:
